src/components/section-display.js

```diff
@@ -215,13 +217,22 @@ function SectionDisplay({
     }
 
     return (
-      <DisplayComponent
-        context={context}
-        content={section.content}
-        input={sectionInput?.data ?? null}
-        canModifyInput={canModifyInput}
-        onInputChanged={onSectionInputChange}
-        />
+      <Fragment>
+        <DisplayComponent
+          context={context}
+          content={section.content}
+          input={sectionInput?.data ?? null}
+          canModifyInput={canModifyInput}
+          onInputChanged={onSectionInputChange}
+          />
+        {!!registeredPlugin.info.allowsInput && !!documentInputId && (
```

Comment:
Maybe also for deleted sections: `&& !section.deletedOn`?

Reply:
Although - that might hide historical comments :thinking:

Reply:
So I think we should keep it as is, please ignore all the comments on this line :smile:

Reply:
What comments? 😵‍💫